app/consumers/recommendation_consumer.rb

```diff
@@ -1,10 +1,10 @@
 class RecommendationConsumer
   include Hutch::Consumer
 
-  consume "copro.recommendations.*"
+  consume "copro.recommendations.*.*"
```

Comment:
curious to know what's the difference between `consume "copro.recommendations.*"` and `consume "copro.recommendations.*.*"`